src/ert/gui/simulation/view/realization.py

```diff
@@ -153,3 +170,26 @@ def _paint_inner_grid(painter: QPainter, rect: QRect, colors) -> None:
 
     def sizeHint(self, option, index) -> QSize:
         return self._size
+
+    def eventFilter(self, watched, event: QEvent):
+        if event.type() == QEvent.Type.ToolTip:
+            mouse_pos = event.pos()
+            parent: RealizationWidget = self.parent()
+            view = parent._real_view
+            model: QAbstractItemModel = view.model()
+            for row in range(model.rowCount()):
+                index = model.index(row, 0)
+                rect = view.visualRect(index)
+                if rect.contains(mouse_pos):
```

Comment:
Is there a better way to get the current item than just loop over all the models?

Reply:
I am not sure, but I will figure it out. Things like this in Qt seem overly complicated.